test/AspNet.Security.OAuth.Providers.Tests/Discord/DiscordTests.cs

```diff
@@ -46,5 +48,76 @@ public async Task Can_Sign_In_Using_Discord(string claimType, string claimValue)
             // Assert
             AssertClaim(claims, claimType, claimValue);
         }
+
+        [Fact]
+        public async Task Authorization_Endpoint_Uri_by_Default_Does_Not_Contain_Prompt()
+        {
+            static void ConfigureServices(IServiceCollection services)
+            {
+                services.PostConfigureAll<DiscordAuthenticationOptions>((options) =>
+                {
+                    options.Events = new OAuthEvents
+                    {
+                        OnRedirectToAuthorizationEndpoint = ctx =>
+                        {
+                            Assert.DoesNotContain("prompt=", ctx.RedirectUri, StringComparison.InvariantCulture);
```

Comment:
In these circumstances, I usually have a `bool` and set it to `true` as part of the callback and then assert on that in the test. Otherwise there's no guarantee the assert even ran.

Reply:
I don't have a clear picture in my head how that would work?  Do you have an example in another test that you could point me towards?  If not, that is okay, I will work towards making this change, but an example would be helpful.  Thanks

Reply:
I had initially tried what you proposed, but given the nature of the static method inside the test, I hit a snag.

Reply:
I just pushed up this change, let me know if it still isn't quite what you had in mind.